contracts/token/SRC20.sol

```diff
@@ -441,6 +336,32 @@ contract SRC20 is ISRC20, ISRC20Owned, ISRC20Managed, SRC20Detailed, Featured,
         return true;
     }
 
+    function transfer(address to, uint256 value) external returns (bool) {
+        if (_restrictions != ITransferRules(0)) {
+            require(_restrictions.authorize(msg.sender, to, value), "Transfer not authorized");
+
+            _restrictions.doTransfer(msg.sender, to, value);
+        } else {
+            _transfer(msg.sender, to, value);
```

Comment:
**Notice**: If SRC20 does not have on-chain rules (`_restrictions == 0`), someone can call this transfer call and avoid off-chain rules.

Reply:
This is intended. When restrictions are not set it is utility token, and if you want just off-chain rules restrictions should be deployed with no rules (because there are no on-chain rules and it should be visible on contract)